### PR TITLE
Added PyRIT RT env script

### PR DESCRIPTION
**Pull request by pramit-d**
https://github.com/Be-Secure/besecure-ce-env-repo/issues/68

### CODE REVIEW

**Review by anilsingla — APPROVED** [review]

File: PyRIT/0.0.1/besman-PyRIT-RT-env.sh [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    if [[ -d $BESMAN_ASSESSMENT_DATASTORE_DIR ]] 
+    then
+        __besman_echo_white "Assessment datastore found at $BESMAN_ASSESSMENT_DATASTORE_DIR"
+    else
+        __besman_echo_white "Cloning assessment datastore from $\BESMAN_USER_NAMESPACE/besecure-assessment-datastore"
+        __besman_repo_clone "$BESMAN_USER_NAMESPACE" "besecure-assessment-datastore" "$BESMAN_ASSESSMENT_DATASTORE_DIR" || return 1
+
+    fi
+    # Please add the rest of the code here for installation
+    bash
```
[the remark this answers is not in the document]
Comment (pramit-d) [reply PyRIT/0.0.1/besman-PyRIT-RT-env.sh]: It was required to update the current terminal session after conda installation. But I have removed bash, it was creating issues for other BeSman operations.

File: PyRIT/0.0.1/besman-PyRIT-RT-env.sh [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    __besman_run_ansible_playbook_extra_vars "$BESMAN_ARTIFACT_TRIGGER_PLAYBOOK_PATH/$BESMAN_ARTIFACT_TRIGGER_PLAYBOOK" "bes_command=remove role_path=$BESMAN_ANSIBLE_ROLES_PATH" || return 1
+    if [[ -d $BESMAN_ARTIFACT_DIR ]]; then
+        __besman_echo_white "Removing $BESMAN_ARTIFACT_DIR..."
+        rm -rf "$BESMAN_ARTIFACT_DIR"
+    else
+        __besman_echo_yellow "Could not find dir $BESMAN_ARTIFACT_DIR"
+    fi
+    # Please add the rest of the code here for uninstallation
+    __uninstall_conda
+    bash
```
[the remark this answers is not in the document]
Comment (pramit-d) [reply PyRIT/0.0.1/besman-PyRIT-RT-env.sh]: Removed

File: PyRIT/0.0.1/besman-PyRIT-RT-env-config.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+BESMAN_ORG: Be-Secure #***
+
+# project/ml model/training dataset
+BESMAN_ARTIFACT_TYPE: project
+
+# Name of the artifact under assessment.
+BESMAN_ARTIFACT_NAME: PyRIT #***
+
+# Version of the artifact under assessment.
+BESMAN_ARTIFACT_VERSION: PyRIT
```
Comment (asa1997) [inline PyRIT/0.0.1/besman-PyRIT-RT-env-config.yaml:19]: why is version pyrit?

File: PyRIT/0.0.1/besman-PyRIT-RT-env.sh [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    unset CONDA_EXE
+    unset _CE_M
+    unset _CE_CONDA
+    unset CONDA_PYTHON_EXE
+    unset CONDA_SHLVL
+    unset CONDA_DEFAULT_ENV
+    unset CONDA_PROMPT_MODIFIER
+    source $HOME/.bashrc
+    source $HOME/.zshrc
+    hash -r
```
Comment (asa1997) [inline PyRIT/0.0.1/besman-PyRIT-RT-env.sh:127]: what is the use of `hash -r`

Comment (pramit-d) [reply PyRIT/0.0.1/besman-PyRIT-RT-env.sh:127]: It is used to clear cached paths related to conda.

File: PyRIT/0.0.1/besman-PyRIT-RT-env-config.yaml (hunk 3 above)
Comment (pramit-d) [reply PyRIT/0.0.1/besman-PyRIT-RT-env-config.yaml]: Updated version to v0.3.0

File: PyRIT/0.0.1/besman-PyRIT-RT-env.sh [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+    __besman_run_ansible_playbook_extra_vars "$BESMAN_ARTIFACT_TRIGGER_PLAYBOOK_PATH/$BESMAN_ARTIFACT_TRIGGER_PLAYBOOK" "bes_command=reset role_path=$BESMAN_ANSIBLE_ROLES_PATH" || return 1
+    # Please add the rest of the code here for reset
+    __uninstall_conda
+    __install_conda
+}
+
+function __install_conda {
+    echo "Checking for Anaconda..."
+    temp_file=$(mktemp)
+    bash -c 'export PATH=$HOME/anaconda3/bin:$PATH; function __check_anaconda {
```
Comment (asa1997) [inline PyRIT/0.0.1/besman-PyRIT-RT-env.sh:108]: @pramit-d Good one!!!

**Pull request merged**